src/pybind/mgr/openattic/module.py

```diff
@@ -4,6 +4,7 @@
 openATTIC mgr plugin (based on CherryPy)
 """
 
+import logging
```

Comment:
The manager already provides the logging infrastructure. We should not instantiate a new logger.